client/src/pages/TaskFormPage/TaskFormPage.js

```diff
@@ -1,40 +1,138 @@
-import React from "react";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
 import styles from "./TaskFormPage.module.css";
 import { InputBox } from "../../components/InputBox/InputBox";
 import { CredentialsButton } from "../../components/CredentialsButton/CredentialsButton.js";
+import { DateSelector } from "../../components/DateSelector/DateSelector.js";
+import { instance } from "../../axios";
 
 const TaskFormPage = () => {
+  const navigate = useNavigate();
+  const routeToCalendar = () => navigate("/calendar");
```

Comment:
Minor comment: This doesn't need to be a function. It can just be stored as a variable (i.e. `const navigate/routeToCalendar = navigate("/calendar")`)

Reply:
Actually no need for the function, we can just use `navigate("/calendar")` directly

Reply:
Directly use `navigate` instead.